velox/functions/prestosql/FromUtf8.cpp

```diff
@@ -223,16 +223,21 @@ class FromUtf8Function : public exec::VectorFunction {
       } else {
         auto flatBase = base->asFlatVector<StringView>();
         auto stringBuffers = flatBase->stringBuffers();
-        localResult = decodedInput.wrap(
-            std::make_shared<FlatVector<StringView>>(
-                context.pool(),
-                VARCHAR(),
-                nullptr,
-                flatBase->size(),
-                flatBase->values(),
-                std::move(stringBuffers)),
-            *input,
-            rows.end());
+
```

Comment:
would it make sense to add a unit test?

Reply:
I'm not sure how to reproduce the error in a unit tests. We somehow need to "corrupt" the memory.